unit test for faspfenics DONE

arthbous opened the pull request:
The test for faspfenics is all good. Please test it and accept the merge.

maximilianmetti commented:
@arthbous 
I made two changes:
- I wrapped the initial and final statements of ``test_faspfenics`` in ``DEBUG`` blocks to make it less verbose
- I added a script that we should keep updating to run all the unit tests at once.  This will be useful to run before each merge into master.

This is ready for merge, and I would merge it myself but wanted to raise one point.  Do you think we should make the error messages and ``DEBUG`` messages follow a consistent format?  I think it would be nice, but it isn't *necessary*.

You can merge into master.

arthbous commented:
I changed the layout of the test_eafe to make it similar to the other one. I also added DEBUG as an argument non need to recompile now.

Now for fun run:
./test_all.sh
or run
./test_all.sh DEBUG

AND TADA!

maximilianmetti commented:
I like it a lot!
